src/js/clinvar/eva-clinical-widget-panel.js

```diff
@@ -241,6 +240,34 @@ EvaClinicalWidgetPanel.prototype = {
             ]
         });
 
+        var conseqTypeTreeStore = Ext.create('Ext.data.TreeStore', {
+            model: 'Tree Model',
+            proxy: {
+                type: 'memory',
+                data:clinvarConseqTypeFilter.consequenceTypes['78'],
+                reader: {
+                    type: 'json'
+                }
+            },
+            root: {
+                expanded: false
+            }
+        });
+
+        clinvarConseqTypeFilter.panel.reconfigure(conseqTypeTreeStore);
+
+        var nodes = clinvarConseqTypeFilter.panel.getRootNode()
+        nodes.cascadeBy(function (n) {
+            if (n.isLeaf()) {
+                n.data.qtip =  n.data.description;
+            }
+        });
+
+        if (!_.isEmpty(_this.selectSO)) {
```

Comment:
Where is `selectSO` initialized? It doesn't seem to be looking just at this file.